cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/ScheduledRunTimeTest.java

```diff
@@ -74,7 +75,11 @@ public void after() throws Exception {
       });
   }
 
+  /*
+   * TODO : to fix after CDAP-19775 is addressed
```

Comment:
so we have /version endpoints that call schedules, preferences etc - these are being called with -SNAPSHOT in the tests - this is how a user might currently call these endpoints (since they are unaware of the app created latest version) - we must change these api behavior to ignore the user provided version and map it to the latest instead of changing the tests - this is the discussion we had with @tivv yesterday